alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/NodeMetrics.java

```diff
@@ -0,0 +1,82 @@
+package eu.xenit.alfred.telemetry.binder;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+
+import java.util.concurrent.TimeUnit;
+import java.util.function.ToDoubleFunction;
+
+import io.micrometer.core.instrument.TimeGauge;
+import org.alfresco.repo.domain.node.NodeDAO;
+import org.alfresco.repo.domain.permissions.AclDAO;
+import org.alfresco.repo.transaction.RetryingTransactionHelper;
+import org.alfresco.service.transaction.TransactionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NodeMetrics {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NodeMetrics.class);
+
+    public static final String ACL_PREFIX = "alfresco.alc";
```

Comment:
`.alc` :-)

Reply:
which shows you need a unit test :-)

Reply:
Wow. I accidently clicked on "ask for another review." Sorry. I'll fix it.

Reply:
@tgeens How do I add a test that is useful. I've been trying to figure out how I can check if my NodeMetrics works correctly. 

I've tried two things but they turned out to be not as expected and useless. Any guidance to correct documentation? Some hints?